Firmware/Marlin_main.cpp

```diff
@@ -4484,6 +4484,7 @@ void process_commands()
       }
       st_synchronize();
       delay_keep_alive(codenum);
+      previous_millis_cmd.start();
```

Comment:
Interesting this commit didn't have effect on code size 🤔 is this being optimised away?